models/list/list.go

```diff
@@ -113,10 +116,10 @@ func (u *ListStruct) Save(repo *repository.RepositoryStruct, OauthToken *sql.Nul
 	}
 	tx.Commit()
 	u.ID, _ = result.LastInsertId()
-	return true
+	return nil
 }
 
-func (u *ListStruct) Update(repo *repository.RepositoryStruct, OauthToken *sql.NullString, title *string, color *string, action *string) bool {
+func (u *ListStruct) Update(repo *repository.RepositoryStruct, OauthToken *sql.NullString, title *string, color *string, action *string) (e error) {
```

Comment:
__[golint]__
* [golint] exported method ListStruct.Update should have comment or be unexported